source/mir/glas/common.d

```diff
@@ -0,0 +1,66 @@
+/++
+$(H2 General Matrix-Vector Multiplication)
+
+$(SCRIPT inhibitQuickIndex = 1;)
+
+This is a submodule of $(LINK2 mir_glas.html, mir.glas).
+
+License: $(LINK2 http://boost.org/LICENSE_1_0.txt, Boost License 1.0).
+
+Authors: Ilya Yaroshenko
++/
+module mir.glas.common;
+
+/// Conjugation type
```

Comment:
you might not like it, but I think API descriptions should always expect zero or little knowledge of the user and try to explain a term